CodeMeKathy/glowing-memory
# Feature: As a user, I want to read the names of all of the items in my shopping list

## Description


This Pull Request (PR) features data from the cloud firebase database passed through to the `<App />`, and `<List />`, and  `<ListItems />` components with minimal initial new styling to better align with the provided design spec mockup.

## Related Issue


Closes #11 

## Acceptance Criteria


## Type of Changes


|     | Type                       |
| --- | -------------------------- |
|     | :bug: Bug fix              |
| ✅   | :sparkles: New feature     |
| ✅     | :hammer: Refactoring       |
|     | :100: Add tests            |
|   | :link: Update dependencies |
|     | :scroll: Docs              |

## Updates

### Before


<details markdown=block>
<summary markdown=span>Design Spec Mockup</summary>

- Shopping List (`<List/>`):
![image](https://user-images.githubusercontent.com/32943879/228936451-9f509cd4-e62c-41d9-bf44-99631662407b.png)
</details>

<details markdown=block>
<summary markdown=span>Initital Shopping List UI Layout</summary>

- Shopping List (`<List/>`):
![image](https://user-images.githubusercontent.com/32943879/228940137-cffa0016-de17-48c7-badf-d87c1ccd86b1.png)
</details>

### After


<details markdown=block>
<summary markdown=span>UI Revisions</summary>

- Shopping List `<List/>_Empty_List`):
![`<List/>_Empty_List`](https://user-images.githubusercontent.com/32943879/228937468-d3fb7137-75a8-4505-9957-65511cc5968e.png)

- Shopping List `<List/>_Items`):
![`<List/>_Items`](https://user-images.githubusercontent.com/32943879/228937645-ad261309-3ad0-4b7f-8bca-2c0855e4a206.png)
</details>

## Testing Steps / QA Criteria
 

- From your terminal, run `git checkout kl-view-shopping-list`.
- Once on the `kl-view-shopping-list` branch in the browser, visit the new `<List/>` component (http://localhost:3000/list).
- A rendered Smart shopping list with a single item of "Paper Towels" should be detailed.
